demos/python_demos/whiteboard_inpainting_demo/README.md

```diff
@@ -0,0 +1,76 @@
+# Whiteboard inpainting demo
+
+This demo demonstrates how to disable a person from a whiteboard and show
```

Comment:
What do you mean by disabling a person? :astonished:

Reply:
The description updated.